extensions/typescript-language-features/src/languageFeatures/codeLens/implementationsCodeLens.ts

```diff
@@ -25,7 +25,9 @@ export default class TypeScriptImplementationsCodeLensProvider extends TypeScrip
 		super(client, _cachedResponse);
 		this._register(
 			vscode.workspace.onDidChangeConfiguration(evt => {
-				if (evt.affectsConfiguration(`${language.id}.implementationsCodeLens.showOnInterfaceMethods`)) {
+				if (evt.affectsConfiguration(`${language.id}.implementationsCodeLens.showOnInterfaceMethods`) ||
+					evt.affectsConfiguration(`${language.id}.implementationsCodeLens.showOnClassMethods`))
```

Comment:
This new setting needs to be documented in the `package.json` too. Look at how `showOnInterfaceMethods` is implemented for an example